modules/ROOT/pages/lb-mapping-rules.adoc

```diff
@@ -8,165 +8,278 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
+To determine the load balancer ID, perform a `GET` request to your endpoint:
 
-By default, your load balancer listens to external requests on HTTPS and communicates internally with your worker over HTTP. If you configured your Mule application within the VPC to listen on HTTPS, make sure you set `upstreamProtocol` to `HTTPS` when creating the mapping list using the xref:anypoint-platform-cli.adoc#cloudhub-load-balancer-mappings-add[`load-balancer mappings add`] command.
+[source,text,linenums]
+----
+/organizations/{orgid}/loadbalancers
+----
 
-=== URL Mapping
 
-You can pass the app name as an input URI and map it directly to the app name in CloudHub:
+== Using a Literal 1:1 Mapping
+
+If you have only one application, you can map the literal app name.
+In this case, the rule must match the application exactly. 
+
+This rule maps the default load balancer `lb-demo.lb.anypointdns.net` directly to the `myApp.cloudhub.io` app in CloudHub:
+
+[%header,cols="10a,20a,20a,20a,10a"]
+|===
+| Rule # | *Input Path* | *Target App* | *Output Path* | *Protocol*
+| 1 | / | myApp | / | http 
+|===
+
+This rule matches the `dev-misc-mythellotest-v1-myapp` app:
 
-[%header,cols="10a,20a,20a,10a"]
+[%header,cols="10a,20a,20a,20a,10a"]
 |===
-|Rule # |Input URL 2+^| Output URL
-|   |    *URI*   |       *appName*   |   *appURI*
-| 0 | /{app}/    | {app}             | /
+| Rule # | *Input Path* | *Target App* | *Output Path* | *Protocol*
+| 1 | /{layera}/{domaina}/v{versiona}/ | dev-{layera}-{domaina}-{versiona}-myapp | /api/ | http
 |===
 
-This rule maps `lb-demo.lb.anypointdns.net/{app}` to `{app}.cloudhub.io`. `{app}` is a pattern for any application name you pass in the inbound request.
 
+== Using Patterns in Mapping Rules
 
-=== Host Mapping
+A _pattern_ is a string that defines a template for matching an input text. The value in curly brackets (`{   }`) is treated as a variable.
 
-If you have a wildcard certificate (such as `*.example.com`), you can use the `subdomain` variable to map any subdomain:
+* Variable *names* can contain only lowercase letters (`a-z`) and no other characters.
+* Variable *values* can contain the characters `a-z0-1.&?-_`, but can't contain slash characters (`/`).
 
-[%header,cols="10a,20a,20a,10a"]
+
+[IMPORTANT]
+DLBs don't support patterns in the *Output Path* (`appURI`).
+
+If you set up a DNS CNAME record to map `example.com` to `_lb-name_.lb.anypointdns.net`,
+you can then use a mapping rule to map `app.example.com` to a different deployed CloudHub Mule application name.
+
+[%header,cols="10a,20a,20a,20a,10a"]
 |===
-|Rule # |Input URL 2+^| Output URL
-|   |  *URI*   |       *appName*    |  *appURI*
-| 0 | /        | {subdomain}   | /
+| Rule # | *Input Path* | *Target App* | *Output Path* | *Protocol*
+|1 | /api/v1/ | default-app | /api/v1/ | http
+|2 | /api/v1/secondapp | second-app | /api/v1	http
```

Comment:
Don't know why but when I change to rich diff mode instead of source diff, I can't see this rule

Reply:
There's a bug in the table (missing pipe on the second rule before http). Fixed.